src/tests/ftest/container/boundary.py

```diff
@@ -22,120 +22,142 @@ class BoundaryTest(TestWithServers):
+
         Args:
             pool(str): pool handle.
-            pool_num (int): pool number to create containers.
             num_containers (int): number of containers to create.
-            with_io (bool): enable container test with execute_io.
         """
-
-        self.log.info("==(2.%d)create_containers start.", pool_num)
+        self.log.info("==(2.%s)create_containers start.", pool.label)
         thread_manager = ThreadManager(self.create_container_and_test, self.timeout - 30)
 
         for cont_num in range(num_containers):
-            thread_manager.add(
-                pool=pool, pool_num=pool_num, cont_num=cont_num, with_io=with_io)
+            thread_manager.add(pool=pool, cont_num=cont_num)
 
         # Launch the create_container_and_test threads
         self.log.info("==Launching %d create_container_and_test threads", thread_manager.qty)
         failed_thread_count = thread_manager.check_run()
         self.log.info(
-            "==(2.%d) after thread_manager_run, %d containers created.", pool_num, num_containers)
+            "==(2.%s) after thread_manager_run, %d containers created.", pool.label, num_containers)
         if failed_thread_count > 0:
             msg = "#(2.{}) FAILED create_container_and_test Threads".format(failed_thread_count)
             self.d_log.error(msg)
             self.fail(msg)
 
-    def create_pools(self, num_pools=10, num_containers=100, with_io=False):
+    def create_pools(self, num_pools, num_containers):
         """To create number of pools and containers parallelly.
+
         Args:
             num_pools (int): number of pools to create.
             num_containers (int): number of containers to create.
-            with_io (bool): enable container test with execute_io.
-        """
-
-        # Setup the thread manager
-        thread_manager = ThreadManager(self.create_containers, self.timeout - 30)
-
-        for pool_number in range(num_pools):
-            pool = self.get_pool()
-            thread_manager.add(
-                pool=pool, pool_num=pool_number, num_containers=num_containers, with_io=with_io)
-            self.log.info("=(1.%d) pool created, %d.", pool_number, pool)
 
-        # Launch the create_containers threads
-        self.log.info("=Launching %d create_containers threads", thread_manager.qty)
-        failed_thread_count = thread_manager.check_run()
-        if failed_thread_count > 0:
-            msg = "#(1.{}) FAILED create_containers Threads".format(failed_thread_count)
-            self.d_log.error(msg)
-            self.fail(msg)
+        """
+        # Create pools in parallel
+        pool_manager = ThreadManager(self.get_pool, self.timeout - 30)
+        for _ in range(num_pools):
+            pool_manager.add()
+        self.log.info('Creating %d pools', num_pools)
+        num_failed = pool_manager.check_run()
+        if num_failed > 0:
+            self.fail('{} pool create threads failed'.format(num_failed))
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string